tsconfig.json

```diff
@@ -2,7 +2,7 @@
   "compilerOptions": {
     "target": "es5",
     "module": "esnext",
-    "strict": true,
+    "strict": false,
```

Comment:
had problems impprt from vue3-highlightjs, because it has no type. Check newest commit, changed back to strict but added noImplicitAny to false which also solves this problem.